frontends/web/src/routes/account/send/components/inputs/receiver-address-input.tsx

```diff
@@ -14,47 +14,67 @@
  * limitations under the License.
  */
 
-import { ChangeEvent, useCallback, useContext, useState } from 'react';
+import { ChangeEvent, useCallback, useContext, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useMediaQuery } from '@/hooks/mediaquery';
+import * as accountApi from '@/api/account';
 import { getReceiveAddressList } from '@/api/account';
 import { debug } from '@/utils/env';
+import { ReceiverAddressWrapper } from './receiver-address-wrapper';
+import { QRCodeLight, QRCodeDark } from '@/components/icon';
 import { DarkModeContext } from '@/contexts/DarkmodeContext';
 import { Input } from '@/components/forms';
-import { QRCodeLight, QRCodeDark } from '@/components/icon';
+import { useMediaQuery } from '@/hooks/mediaquery';
 import { ScanQRDialog } from '@/routes/account/send/components/dialogs/scan-qr-dialog';
 import style from './receiver-address-input.module.css';
 
-type TToggleScanQRButtonProps = {
-    onClick: () => void;
-}
-
 type TReceiverAddressInputProps = {
-    accountCode?: string;
+    account?: accountApi.IAccount;
+    activeAccounts?: accountApi.IAccount[];
     addressError?: string;
     onInputChange: (value: string) => void;
-    recipientAddress: string;
+    onAccountChange?: (account: accountApi.IAccount | null) => void;
     parseQRResult: (uri: string) => void;
+    recipientAddress: string;
+}
+type TToggleScanQRButtonProps = {
+  onClick: () => void;
+  withDropdown?: boolean;
 }
 
-export const ScanQRButton = ({ onClick }: TToggleScanQRButtonProps) => {
+export const ScanQRButton = ({ onClick, withDropdown = false }: TToggleScanQRButtonProps) => {
   const { isDarkMode } = useContext(DarkModeContext);
   return (
-    <button type="button" onClick={onClick} className={style.qrButton}>
+    <button type="button" onClick={onClick} className={`
+     ${style.qrButton || ''}
+     ${withDropdown ? style.withDropdown || '' : ''}`
+    }>
       {isDarkMode ? <QRCodeLight /> : <QRCodeDark />}
     </button>);
 };
 
 export const ReceiverAddressInput = ({
-  accountCode,
+  account,
+  activeAccounts,
   addressError,
   onInputChange,
+  onAccountChange,
   recipientAddress,
-  parseQRResult,
+  parseQRResult
 }: TReceiverAddressInputProps) => {
   const { t } = useTranslation();
   const isMobile = useMediaQuery('(max-width: 768px)');
   const [activeScanQR, setActiveScanQR] = useState(false);
+  const accountCode = account?.code;
+
+  const accountsForReceiverDropdown = useMemo(() =>
+    activeAccounts?.filter(acc =>
+      acc.coinCode !== 'sepeth' &&
+      acc.coinCode !== 'eth' &&
```

Comment:
tested servewallet-mainnet + webdev

ETH is excluded on purpose correct?

but erc20 tokens are still showing up in the dropdown, I assume those should be excluded as well?

Please mention in the commit message what the intention is or whether this is intended behavior.